nano/node/scheduler/bucket.cpp

```diff
@@ -35,7 +35,10 @@ void nano::scheduler::bucket::pop ()
 
 void nano::scheduler::bucket::push (uint64_t time, std::shared_ptr<nano::block> block)
 {
-	queue.insert ({ time, block });
+	const uint64_t bitmask = ~0b11111;
+	uint64_t rounded_time = time & bitmask;
```

Comment:
For readability I would strongly suggest to keep a normal division. Our bottleneck is IO which is many orders of magnitude slower anyway, this division is noop when it comes to performance